antora.yml

```diff
@@ -27,6 +27,8 @@ asciidoc:
     hosted-che-docs: xref:hosted-che:hosted-che.adoc[]
     identity-provider-id: keycloak
     identity-provider: Keycloak
+    identity-provider-legacy-id: keycloak
+    identity-provider-legacy: Keycloak
```

Comment:
@tolusha Can you clarify the expected values?

Reply:
The values about are correct